crowbar_framework/app/controllers/api/crowbar_controller.rb

```diff
@@ -54,6 +77,24 @@ def maintenance
 
   api :GET, "/api/crowbar/repocheck", "Sanity check for Crowbar server repositories"
   api_version "2.0"
+  example '
```

Comment:
@jsuchome @santiph as mentioned below, we may have to define some view to show extra information about response parameters that are unclear. but please in a separate small PR